docs/user/appendix/changelog.md

```diff
@@ -0,0 +1,1092 @@
+# Unreleased changes
```

Comment:
Is there a part of the release process that will update this file? Or are we expected to keep it up to date manually?

Reply:
Hm, well actually this file should not even be here. This is me using `git add --all` again... Good catch though! (Changelog files are copied to the `changelog/` folder everytime we run `made docs`.)